docs/deploy-and-configure/configuration/keycloak/index.md

```diff
@@ -14,79 +14,159 @@ This page documents important steps in order to configure Keycloak as an authent
-      - Go to `Roles` and add the `elds-admins` role
-      - Go to `Service Account Roles -> Client Roles (cmem-service-account)` and add the `elds-admins` role to `Assigned Roles`
-      - Go to `Mappers`
-          - Click `Create`
-            - Name `roles`
-            - Mapper Type `User Client Role`
-            - Client ID `cmem-service-account`
-            - Token Claim Name `groups`
-            - Enable `Add to access token`
-          - (Only for DP < 19.10.1) Click `Create`
-            - Name `DataPlatform audience`
-            - Mapper Type `Audience`
-            - Included Client Audience --> "Select One" (do not touch it)
-            - Included Custom Audience `dataplatform`
-            - Disable `Add to ID token`
-            - Enable `Add to access token`
+There are two (three) different kinds of clients used by Corporate Memory. One client is used by DM/DP/DI to authenticate a user for using the UI. The other client is for using the command line client as technical user. Depending on the environment there might be an other use case when running background schedules, then a third client, also as technical user, might be useful.
+
+### Add a client by importing
+
+Add a client called `cmem` by select clients, then create client. The client described below can also be imported. Please download file below, then select "Import client". For `cmem-service-account` you have to edit the file an replace the secret or regenerate the secret in keycloak after the import. Files:
+
+  - [client configuration for using the ui](cmem.json)
+  - [client configuration with credentials for technical account](cmem-service-account.json)
+
+
+![Dialog import cmem client](import-client-cmem.png)
+
+### Add a client manually (Login into UI)
+
+This client is intended for usage by DataManager/Dataplatform and DataIntegration (user login):
+
+![Dialog create cmem client](createClient_1.png)
+
+  - Client type: OpenID Connect
+  - Client ID: i.e. `cmem`, you need to remember this and use this later
+  - Name and Description: fill as you like
+  - Select 'Next'
+  - Client authentication: Off
+  - Authorization: Off
+  - until v22.2:
+    - Enable `Standard Flow Enabled` (enables OAuth 2.0 Authorization Code Flow)
+    - Enable `Implicit Flow Enabled`
+  - from v23.1:
+    - Enable `Standard Flow Enabled` (enables OAuth 2.0 Authorization Code Flow)
+  - Save
+
+![Dialog create cmem client](createClient_2.png)
+
+The dialog above closes and you land on the configuration sections of this client:
+
+  - Valid redirect URIs: Add the adequate URL pattern (wildcard `http://example.org/*` works) to `Valid Redirect URIs` (`*` for testing purposes is acceptable)
+  - Switch the Tabs to 'Client scopes' and select the first scope (i.e.: cmem-service-account-dedicated)
+
+![Dialog select cmem-service-account-dedicated](createClient_11.png)
+![Dialog create mapper](createClient_4.png)
+![Dialog create mapper](createClient_5.png)
+
+  - `Configure a new mapper`
```

Comment:
Where is this `Configure a new mapper` option? I cannot find it ... maybe add a short note here where to find it.